sinks/splunk/splunk.go

```diff
@@ -197,6 +198,13 @@ func (sss *splunkSpanSink) makeHTTPRequest(req *http.Request) {
 	}()
 
 	resp, err := sss.httpClient.Do(req)
+	if uerr, ok := err.(*url.Error); ok && uerr.Timeout() {
```

Comment:
This is an interesting Go… thing. :)

Reply:
yuuuup, that type assertion was a bit too complex to let it go in untested... but after a bit of staring at it, I think I understand it for real. Wish we could use contexts with timeouts there, but since the request is already in `Client.Do` while the body is being written, I couldn't come up with a reasonable way to represent that.

Reply:
Asking errors how they behave! :D